assets/video/test0.srt

```diff
@@ -59,46 +59,50 @@ AI is capable of converting spoken language into text and translating it into ot
 AI is a central component of robotics.
 
 16
-00:01:20,000 --> 00:01:26,000
+00:01:20,000 --> 00:01:25,000
```

Comment:
I think you pushed this file by mistake.
The assets folder is for testing purposes only.
Please add it to `gitignore`

Reply:
Yes, I pushed it by mistake, but only discovered that after issuing the pull request. It had been better if I added some explanation. Sorry.
AFAIK adding to gitignore does not help anymore once it is part of the repository. But I won't commit it anymore if I generate it again.

Reply:
No worries. 
You can still add it to `gitignore` just to avoid future commits.